playbooks/byo/openshift-checks/roles

```diff
@@ -0,0 +1 @@
+../../../roles
```

Comment:
Yes, `common` already has a symlink. I think back when I started this I was exploring how to have less layers to get something started, but definitely we should follow a single pattern.